tests/integration/distribution/keeper/msg_server_test.go

```diff
@@ -1057,6 +1057,9 @@ func TestCannotDepositIfRewardPoolFull(t *testing.T) {
 	)
 	assert.NilError(t, err)
 
+	// transfer tokens from distribution module back to the account for the second deposit
+	assert.NilError(t, f.bankKeeper.SendCoinsFromModuleToAccount(f.sdkCtx, distrtypes.ModuleName, sdk.AccAddress(operatorAddr), maxCoins))
```

Comment:
this test started failing after the overflow protection was added in https://github.com/celestiaorg/cosmos-sdk/pull/651

since this account is no longer claiming rewards, it has no funds to delegate. Therefore if we give it more funds now it does, which fixes the failure.

Reply:
the first thing that failed was creating a validator, which was fixed in some other PR, then we got this failure, which is fixed now.

Reply:
> this test started failing after the overflow protection was added in #651

Tests for the distribution/keeper module passed on that last commit for that PR: https://github.com/celestiaorg/cosmos-sdk/actions/runs/16147262177/job/45569353954#step:5:35